index.html

```diff
@@ -231,5 +231,97 @@ <h3 class="proj__title">Skills</h3>
         </article>
       </div>
     </main>
+
+    <footer class="contact__form">
+      <div class="contact__title">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
```

Comment:
Kindly stick to the font color given in the design, the color of your test is looking different from the design🤗
your font
<img width="255" alt="your_____font" src="https://user-images.githubusercontent.com/64643995/213020940-ddf39900-a177-4b15-924b-6c937403c4ba.png">

design font
<img width="272" alt="design_____font" src="https://user-images.githubusercontent.com/64643995/213020971-ddd01125-05af-49ae-8b36-f5e55f59cc9d.png">

Reply:
Thanks, for the feedback.